generators/app/index.js

```diff
@@ -258,7 +258,7 @@ module.exports = generators.Base.extend({
           this.disableLogger = answers.disableLogger ? answers.disableLogger :  this.options.disableLogger || 'false';
           this.noMinify = answers.noMinify ? answers.noMinify :  this.options.noMinify || 'false';
           this.enableValidator = this.options.enableValidator || 'false';
-          this.htmlValidator = this.options.enableValidator || '{validator: "http://html5.validator.nu", format: "text", suppressWarnings: true}';
+          this.htmlValidator = this.options.enableValidator || '{"validator": "http://html5.validator.nu", "format": "text", "suppressWarnings": true}';
```

Comment:
it may be a bit hard to follow but, for example:

```js
port: process.env.HTTP_PORT || process.env.NODE_PORT || params.port || pkg.rooseveltConfig.port || 43711
```

in the above example, the default is the last condition of the `or` chain, so, the default would be 43711.